README.md

```diff
@@ -1,2 +1,45 @@
-# strike-off-objections-web
+# Strike Off Objections Web
+
+### Overview
+
 Web frontend for submitting objections to a company being struck off the register.
+
+### Requirements
+
+In order to run the service locally you will need the following:
+
+- [NodeJS](https://nodejs.org/en/)
+- [Homebrew](https://formulae.brew.sh/formula/node)
+- [Git](https://git-scm.com/downloads)
+
+### Getting started
+
+To checkout and build the service:
+1. Clone [Docker CHS Development](https://github.com/companieshouse/docker-chs-development) and follow the steps in the README. 
+2. Run ./bin/chs-dev modules enable strike-off-objections
+3. Run ./bin/chs-dev development enable strike-off-objections (this will allow you to make changes).
```

Comment:
`.bin/chs-dev development enable strike-off-objections-web`

Reply:
See above

Reply:
Not sure what you mean by see above?
strike-off-objections is not a valid service to enable development mode, strike-off-objections is a module, the service tbe user needs to enable is strike-off-objections-web

Reply:
Just missing web off the end.